.github/pull_request_template.md

```diff
@@ -46,16 +46,10 @@ Leave some tips for your reviewer, like:
 Delete this section if no tips.
 -->
 
-### Testing
+### Checklist
 
 - [ ] This PR has adequate test coverage / QA involvement has been duly considered.
-
-### Backward compatibility
-
 - [ ] This PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way) and therefore is tagged with a `T-protobuf` label.
-
-### Release notes
-
-This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
+- [ ]This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
```

Comment:
nit: this lacks the space after the `[ ]` from the previous items.

Reply:
D'oh, thanks.